codegen-v2/manifest/TWCosmosProto.yaml

```diff
@@ -0,0 +1,8 @@
+name: TWCosmosProto
+protos:
+- TW_Cosmos_Proto_Amount
+- TW_Cosmos_Proto_Fee
+- TW_Cosmos_Proto_Height
+- TW_Cosmos_Proto_Message
+- TW_Cosmos_Proto_SigningInput
+- TW_Cosmos_Proto_SigningOutput
```

Comment:
are those things rly needed for proto?

Reply:
I don't have a full overview of the build system yet, I don't know (maybe @hewigovens can comment), of of those are just an alias for `TWData *_Nonnull`, anyway. If we do need those, we should probably put them all into a single document.

Reply:
That's very verbose, yeah curious to know what @hewigovens thinks about it

Reply:
Those are aliases generated by `protobuf-plugin` (see `c_typedef.cc`), not really needed if the new codegen can support converting serialized protobuf model bytes from to equivalent Swift `Struct` or Kotlin `Class`